modules/global-classes/usage/applied-global-classes-usage.php

```diff
@@ -54,44 +58,35 @@ private function get_classes_count_per_element_type( $elements_data, $global_cla
 			}
 
 			/** @var Atomic_Element_Base | Atomic_Widget_Base $element_instance */
-			$classes_count = $this->get_classes_count_for_element( $element_instance->get_props_schema(), $element_data, $global_class_ids );
+			$applied_classes_per_element = $this->get_applied_global_classes_per_element( $element_instance->get_props_schema(), $element_data, $global_class_ids );
 
-			if ( 0 !== $classes_count ) {
-				$count_per_type[ $element_type ] ??= 0;
-				$count_per_type[ $element_type ] += $classes_count;
+			foreach ( $applied_classes_per_element as $global_class_id => $count ) {
+				$count_per_class[ $global_class_id ] ??= 0;
+				$count_per_class[ $global_class_id ] += $count;
```

Comment:
```suggestion
				$count_per_class[ $global_class_id ] ??= $count;
```
can't we just do this?

Reply:
No, because we are adding it and not setting it